scripts/migrate.sh

```diff
@@ -18,13 +18,14 @@ solana withdraw-from-vote-account test-ledger/vote-account-keypair.json \
 # create instance
 ./tests/deploy_test_solido.py --verbose
 
+cp ../solido_test.json ../solido_config.json
 # start maintainer
-./target/debug/solido --config ../solido_test.json \
-                      --keypair-path ../solido_old/tests/.keys/maintainer.json \
+./solido_v1/target/release/solido --config ./solido_config.json \
```

Comment:
Why did you rename the directory? Now it's not convenient to develop

Reply:
1. The previous folder names were poorly chosen and confused the users of the scripts. The correct framework is to assign unique names.
2. It's not good practice to use debug builds in user scripts. All user scripts must target on the release build. We can create another script with a _debug postfix to indicate to users that this script is a helper for developers.